Set viewer preferences method support

YunusEmreNalbant opened the pull request:
Added setViewerPreferences method

YunusEmreNalbant committed:
Added `setViewerPreferences` method.

This commit adds a new method `setViewerPreferences` to the `EasyPdf` class, allowing the user to set viewer preferences for the PDF document. The method accepts an array of preferences and updates the internal PDF viewer preferences accordingly. This enhancement provides more control over how the PDF is displayed to viewers.

YunusEmreNalbant committed:
Update README.md

frkcn commented on README.md:
````diff
@@ -122,6 +122,11 @@ You can set footer font using `setFooterFontSize()` method.
+You can set the viewer preferences dictionary, which controls how the document is presented on screen or in print, using `setViewerPreferences`.
+```php
+// This will set the print to single sided. 
+$pdf->setViewerPreferences(array('Duplex' => 'Simplex'));
````
We are not using `array('Duplex' => 'Simplex')` syntax anymore, it's need to be change to `['Duplex' => 'Simplex']`.

frkcn replied:
Also adding other duplex preferences as examples will be suitable for users in the documentation.

YunusEmreNalbant replied:
fixed @frkcn

YunusEmreNalbant committed:
Refactoring `setViewerPreferences` to `setDuplex` in `EasyPdf` class.

This commit refactors the method `setViewerPreferences` in the `EasyPdf` class to a more specific `setDuplex` method. Instead of accepting an array of preferences, the new method now takes a single string parameter `name`, which represents the duplex attribute for the PDF document. The method updates the internal viewer preferences with the specified duplex attribute value.

YunusEmreNalbant committed:
Apply style

YunusEmreNalbant committed:
Apply style

YunusEmreNalbant committed:
Apply style

YunusEmreNalbant committed:
The test `it_can_convert_html_to_pdf` updated with setDuplex usage.

This commit updates the `it_can_convert_html_to_pdf` test within the PdfTest class to include an example usage of the `setDuplex` method from the `EasyPdf` class. The test now sets the duplex attribute to 'Simplex' for the PDF document generated during testing, ensuring that duplex attribute handling is correctly tested within the HTML to PDF conversion scenario.
